### ISSUE TITLE
count-variants not runnable without specifying cosmicdb

### ISSUE THREAD

**Issue by lincoln-harris** [czbiohub/cerebra#110]
`count-variants` module cannot be run without specifying a path to a cosmic database file

this is a relatively easy fix

**Comment by ojziff** [comment]
Yes @lincoln-harris,  the instructions say optional but the help `-h` states required:
```
 --cosmicdb TEXT      path to cosmic db file (.tsv)  [required]
```

Please could you advise which cosmic tsv file is required? There are 18 different tsv's available to download from https://cancer.sanger.ac.uk/cosmic/download

Many thanks, 
Oliver
